webpack/ForemanTasks/Components/TasksDashboard/Components/TasksCardsGrid/Components/StoppedTasksCard/StoppedTasksCard.js

```diff
@@ -44,16 +44,22 @@ const StoppedTasksCard = ({
         {__('Stopped')}
       </Card.Title>
       <Card.Body>
-        <table className="table table-bordered table-striped stopped-table">
-          <thead>
-            <tr>
-              <th />
-              <th>{__('Total')}</th>
-              <th>{getQueryValueText(time)}</th>
-            </tr>
-          </thead>
-          <tbody>{StoppedTable(data, query, time, updateQuery)}</tbody>
-        </table>
+        <React.Fragment>
+          <StoppedTable
+            data={data.results}
+            query={query}
+            time={time}
+            updateQuery={updateQuery}
+          />
+          <OtherInfo
+            updateQuery={updateQuery}
+            otherCount={data.other}
+            active={
+              query.state === STOPPED &&
+              query.result === TASKS_DASHBOARD_AVAILABLE_QUERY_RESULTS.OTHER
```

Comment:
Can you just pass the `query` to the component as you did on line 50? And calculate `active` there?

Reply:
Done